packages/ra-ui-materialui/src/input/ReferenceInput.tsx

```diff
@@ -123,10 +115,11 @@ ReferenceInput.defaultProps = {
     page: 1,
     perPage: 25,
     sort: { field: 'id', order: 'DESC' },
+    children: <AutocompleteInput />,
```

Comment:
Isn't a good opportunity to avoid `defaultProps`? We could initialize those defaults when destructuring the props

Reply:
nice to have